README.md

```diff
@@ -9,11 +9,14 @@ Automatically dump and archive PostgreSQL backups to Amazon S3
 ## Setup
 
 Add secrets.conf:
-  - PG_HOST and PG_USER with your PostgreSQL hosts and backup user.
-  - PG_DB with pg connection string
-  - S3_PATH with your Amazon S3 bucket and path
-  - AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY with your aws credentials
-  - MAX_DAYS number of days before backups deleted
+  - PG_HOST - PostgresSQL host
+  - PG_USER - PostgreSQL backup user
+  - PG_DB - pg connection string
+  - DB_NAME= name of database to backup
```

Comment:
Where is DB_NAME being used in the code?

Reply:
This will be used in `performance-dashboards/helm/performance-dashboards/values.yaml` line 39 (https://github.com/OpenGov/performance-dashboards/blob/javier-tello23/OD-1198/daily-backups-to-s3/helm/performance-dashboards/values.yaml)

Reply:
Right, but this variable addition not make sense in the context of this PR. The scripts do not use this variable inside their code.

Reply:
I see, I pushed the changes